src/frontend/apps/desk/src/components/DropButton.tsx

```diff
@@ -44,8 +51,14 @@ export const DropButton = ({
   const [opacity, setOpacity] = useState(false);
   const [isLocalOpen, setIsLocalOpen] = useState(isOpen);
 
+  const ref = useRef<HTMLDivElement>(null);
+
   useEffect(() => {
     setIsLocalOpen(isOpen);
+
+    if (ref.current) {
+      ref.current[isOpen ? 'focus' : 'blur']();
+    }
   }, [isOpen]);
 
```

Comment:
By doing like bellow, `isOpen` don't have to be mandatory.
You can then remove the changes in `src/frontend/apps/desk/src/features/header/AccountDropdown.tsx`.

```suggestion
  useEffect(() => {
    setIsLocalOpen(isOpen);
  }, [isOpen]);

  useEffect(() => {
    if (ref.current) {
      ref.current[isLocalOpen ? 'focus' : 'blur']();
    }
  }, [isLocalOpen]);
```